metricbeat/magefile.go

```diff
@@ -93,6 +94,27 @@ func Update() error {
 	return sh.Run("make", "update")
 }
 
+// MockedTests runs the HTTP tests using the mocked data inside each {module}/{metricset}/testdata folder.
+// Use MODULE={module_name} to run only mocked tests with a single module.
+// Use GENERATE=true or GENERATE=1 to regenerate JSON files.
+func MockedTests() error {
+	goTestDefaultArgs := mage.DefaultGoTestUnitArgs()
```

Comment:
Once you mutate this it will no longer be the default args, so how about just calling it `params`.

Reply:
Good point